component-overview/examples/tabs/TabGroup-noBreak.jsx

```diff
@@ -0,0 +1,7 @@
+import { Tab, TabGroup } from '@sb1/ffe-tabs-react';
+
+<TabGroup noBreak={true}>
```

Comment:
Burde dokumenteras med aria-controls?

Reply:
@HeleneKassandra Noe jeg skal gjøre eller dere?

Reply:
Den er påkrevd i ts?

Reply:
Hvis det er noe som skal inn i dokumentasjonen så er det oss som må legge det inn per dags dato :)  Jeg er litt usikker på hvordan den aria-controls brukes i dag, om det forsatt er nødvendig? Evt hva vi bør si om det i dokumentasjonen

Reply:
Sånn for å svare helt konkret:  Vi legger det til i dokumentasjonen  :)  Så det kommer utenom denne PR'en

Reply:
Ser at den er påkrevd på Tab og TabButton også er den brukt i eksemplet TabGroup-advanced.jsx, men ikke TabGroup.jsx. Det må kanskje da inn i dokumentasjonen :thinking: